apps/core/src/migration/service/convert-monitor-to-app-definition.ts

```diff
@@ -61,17 +62,31 @@ const convertType = (monitorChartType: string) => {
       return DashboardWidgetType.StatusTimeline;
     case MonitorWidgetType.Table:
       return DashboardWidgetType.Table;
+    case MonitorWidgetType.Kpi:
+      return DashboardWidgetType.Kpi;
+    case MonitorWidgetType.StatusGrid:
+      return DashboardWidgetType.Status;
     default:
       return DashboardWidgetType.XYPlot;
   }
 };
 
 const convertHeight = (height: number) => {
-  return height * appCellPerMonitorSquareHeight;
+  if (height === 0) {
+    return appCellPerMonitorSquareHeight;
+  }
+
+  // Add some space between widgets by subtracting 0.5
+  return height * appCellPerMonitorSquareHeight - 0.5;
```

Comment:
potentially dangerous if `height * appCellPerMonitorSquareHeight < .5`, the value becomes less than zero; introduce a min?

Reply:
Good call, updated to use `Math.max` to handle this